tests/capabilities.js

```diff
@@ -117,8 +127,9 @@ define('tests/capabilities',
                 firefoxOS: false,
                 firefoxAndroid: false,
                 widescreen: function() { return false; },
+                os: {type: ''},
             };
-            assert.equal(capabilities.device_type(caps), 'desktop');
+            assert.equal(capabilities.device_type(caps), 'android-mobile');
```

Comment:
need to update this test name (`'is desktop on other mobile'` is no longer true)